### PR TITLE
Sophia/piechart

### PR DESCRIPTION
**Pull request by ssdrsv**

### CODE REVIEW

**Comment by cartersamj** [comment]
good job

**Comment by andrewcpark** [comment]
I approve

**Comment by cartersamj** [comment]
`<h1> wow i love the work you ahve done </h1>`

File: src/App.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import NavBar from './components/NavBar';
 import ScatterPlotContainer from './components/Charts/ScatterPlot/JSX/ScatterPlotContainer';
 import HistogramContainer from './components/Charts/Histogram/JSX/HistogramContainer';
 import LineChartContainer from './components/Charts/LineChart/LineChartContainer';
+import PieChartContainer from './components/Charts/PieChart/JSX/PieChartContainer';
```
Comment (cartersamj) [inline src/App.jsx:15]: this formatting is satisfying

**Review by jukim98 — APPROVED** [review]
good job on pie

**Review by antonioclimbs — APPROVED** [review]
looks clean

**Pull request merged**